src/modules/standard-guidelines/framework-module/FrameworkController.ts

```diff
@@ -159,7 +187,7 @@ export class FrameworkController implements Controller {
          *      403:
          *        description: FORBIDDEN - User is not whitelisted, please use a whitelisted certificate
```

Comment:
Unless the certificates are common knowledge, can we be more specific on _what_ whitelisted certificate is needed?

Reply:
This is internal so we should not broadcast this.  Only the people who have access will know what this means.